src/fu-util.c

```diff
@@ -3484,6 +3524,16 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 	if (priv->no_unreported_check)
 		return TRUE;
 
+	/* any things we can fix? */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (fwupd_security_attr_get_bios_attr_id(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_value(attr) != NULL) {
```

Comment:
I think you need to compare that the BIOS attr is not already set to this don't you?

Reply:
Yes that was my thought too.  We set the BIOS attr ID and target value no matter what.  The other thing you can do is only do this if the security attribute doesn't have a success flag.

Reply:
> if the security attribute doesn't have a success flag

Doh, we should definitely do that. Fixed.

Reply:
OK.  A few other ideas
* You should check here whether or not it has the config_fw flag.  More below.
* In security attributes drop the config_fw flag if the target and current value are the same.

Then client doesn't need to check the values (or even know them).  It just needs to see if config_fw is set, and if so then offer to fix it.